src/test/java/com/solr/clientwrapper/config/timezone/HibernateTimeZoneIT.java

```diff
@@ -64,7 +64,7 @@ void storeInstantWithZoneIdConfigShouldBeStoredOnGMTTimeZone() {
         SqlRowSet resultSet = jdbcTemplate.queryForRowSet(request);
```

Comment:
**_Strictly avoid_** interfering(at least keep it from committing) with project default codes _unless required or specified by the senior_